nautobot_floor_plan/svg.py

```diff
@@ -88,21 +88,77 @@ def _setup_drawing(self, width, depth):
 
         return drawing
 
-    def _label_text(self, label_text_out, step, seed, label_text_in):
+    def _label_text(self, label_text_out, floor_plan, label_text_in, letters):
```

Comment:
Is `letters` a boolean here? If so, we should probably use a more explicit variable name like `is_letters` or `is_alpha` or something to denote it as such.